src/tango_sdp_subarray/SDPSubarray/SDPSubarray.py

```diff
@@ -88,6 +96,16 @@ class SDPSubarray(Device):
 
     receiveAddresses = attribute(dtype=str, access=AttrWriteType.READ)
 
+    toggleReadCbfOutLink = attribute(dtype=bool,
+                                     access=AttrWriteType.READ_WRITE,
+                                     doc='Feature toggle to read '
+                                         'the CSP CBF output link map when '
+                                         'evaluating receiveAddresses.')
+
+    toggleConfigDb = attribute(dtype=bool, access=AttrWriteType.READ_WRITE,
+                               doc='Feature toogle to read and write from '
+                                   'the SDP Config DB')
+
```

Comment:
Is putting feature toggles on the TANGO device the right approach? I would have used environment variables so this can be toggled from the Helm deployment.

Reply:
Yes i expect you are right, this was added as a quick hack. I'll change this right away.